helper/pdfGeneration.js

```diff
@@ -1086,6 +1086,13 @@ exports.assessmentPdfGeneration = async function assessmentPdfGeneration(
     submissionId=""
   ) {
     return new Promise(async function (resolve, reject) {
+
+ //Check if the 'tmp' directory exists or create it if not
+   const tmpDir = path.join(__dirname+"/../", 'tmp');
```

Comment:
@praveenKDass please to resolve the path ( path.resolve)

Reply:
changed @VISHNUDAS-tunerlabs